src/github.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
+const fetch = require('make-fetch-happen');
+
+module.exports = class GithubAPI {
+  cacheDir;
+  auth;
+
+  constructor(config = {}) {
+    config.cacheDir = '.changelog';
+    this.cacheDir = config.cacheDir && path.join(process.cwd(), config.cacheDir, "github");
+    this.auth = this.getAuthToken();
+    if (!this.auth) {
+      throw new Error("Must provide GITHUB_AUTH as a ENV variable");
+    }
+  }
+
+  async getIssueData(repo, issue) {
+    return this._fetch(`https://api.github.com/repos/${repo}/issues/${issue}`);
+  }
+
+  async _fetch(url) {
```

Comment:
why the `_`? underscored things are fully public; if you want it to be private, you could do:
```suggestion
  #fetch = async function fetch(url) {
```

Reply:
yea good point 🤔 not only that, it was only used exactly once... so I just folded it back into the `getIssueData` function 👍